### PR TITLE
WHAN-37 ResourceMedia: overwrite autoCrop via request attributes

### PR DESCRIPTION
**Pull request by mrozati**
Add support for overwriting auto crop via request attribute 'autoCrop'

### CODE REVIEW

File: media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java [hunk 1]
```diff
@@ -83,6 +83,9 @@
   @RequestAttribute(injectionStrategy = InjectionStrategy.OPTIONAL)
   private String cssClass;
 
+  @RequestAttribute(injectionStrategy = InjectionStrategy.OPTIONAL)
+  private Boolean autoCrop;
```
Comment (stefanseifert) [inline media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java]: please add a documentation in the class javadoc comment above for this new parameter.

File: media/changes.xml [hunk 2]
```diff
@@ -36,6 +36,9 @@
       <action type="fix" dev="bkalbitz" issue="WHAN-35">
         Avoid duplicated storing for fileReference in File Upload component.
       </action>
+      <action type="update" dev="mrozati" issue="WHAN-37">
```
Comment (stefanseifert) [inline media/changes.xml]: should by type="add"

File: media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java (hunk 1 above)
Comment (stefanseifert) [reply media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java]: https://github.com/wcm-io/wcm-io-handler/blob/develop/media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java#L57

Comment (mrozati) [reply media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java]: Oh I see, will do it, and add also the other missing ones

**Pull request merged**